Vagrantfile

```diff
@@ -2,7 +2,7 @@
 # vi: set ft=ruby :
 
 # Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
-VAGRANTFILE_API_VERSION = '2'
+VAGRANTFILE_API_VERSION = '2'.freeze
```

Comment:
Are you sure this is Vagrant compatible? I disabled the check in the .rubocop.yml file